msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/AcquireTokenSilentIT.java

```diff
@@ -346,7 +346,7 @@ public void acquireTokenSilent_emptyScopeSet(String environment) throws Exceptio
 
     private IConfidentialClientApplication getConfidentialClientApplications() throws Exception {
         String clientId = cfg.appProvider.getOboAppId();
-        String password = cfg.appProvider.getOboAppPassword();
+        String password = cfg.appProvider.getSecret();
```

Comment:
Why this change?

Reply:
I changed that method name just because the 'obo' part of the name seemed misleading: the values were coming from a method called [getSecret()](https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/9683c9d0f9b5a4d28438ac63be7d8f44e2b7be51/msal4j-sdk/src/integrationtest/java/labapi/KeyVaultSecretsProvider.java#L38) which isn't restricted to just OBO-related secrets, and it was also being used in some [acquire token silent tests](https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/9683c9d0f9b5a4d28438ac63be7d8f44e2b7be51/msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/AcquireTokenSilentIT.java#L349) which aren't using OBO either.

However, the AppCredentialProvider class that has that method also has some other fields/methods with 'obo' in the name. I didn't change those since they weren't used in the new CIAM tests, but changing just one feels like it'd be more confusing than leaving them all. In the latest commit I reverted this change, so it's back to just being 'getOboAppPassword()'